### ISSUE TITLE
[bug] Tooltips aren't at the top of UI

### ISSUE THREAD

**Issue by ggallon** [tldraw/tldraw#883]
Hi,

I noticed that the ToolTips aren't on top of other the tools.

![image](https://user-images.githubusercontent.com/3640689/182880503-320fec58-60d7-4709-882b-7a9c69585595.png)

**Comment by fbfdsouza** [comment]
apparently, this is related to a CSS transform attribute that is messing up with the index of the tooltip element... 
![image](https://user-images.githubusercontent.com/15348339/183298173-34b6eb44-ccc0-4794-b5c2-06649033f406.png)

I recommend the reading: https://www.freecodecamp.org/news/4-reasons-your-z-index-isnt-working-and-how-to-fix-it-coder-coder-6bc05f103e6c/#:~:text=Setting%20some%20CSS%20properties%20like%20opacity%20or%20transform%20will%20put%20the%20element%20in%20a%20new%20stacking%20context.

I tried to change the source code locally to try to manipulate it and see how I could help more, but I came across multiple issues if I tried to change it... maybe it is my machine, but this would be different issue.

**Comment by zachy-ho** [comment]
Took a quick look. I think the tooltip is in the individual button's stacking context, and the stacking context of every other button that comes after that one takes precedence. Will try my hand at this 🤔 .

@steveruizok I'm super new here so just wondering why there's difference in how these `ActionButton`s are implemented vs those in `PrimaryTools` (e.g. specific designs for each), i.e.
```typescript
// --- In `ActionButton.tsx` ---
<ToolButton>
  <Tooltip>
    <Icon />
  <Tooltip>
</ToolButton>

// --- In `PrimaryTools.tsx` ---
<ToolButtonWithTooltip>
  <Icon />
</ToolButtonWithTooltip>
// which is the same as
<Tooltip>
  <ToolButton>
    <Icon />
  </ToolButton>
</Tooltip>
```

Just curious if a potential (but probably unlikely) solution to this is to make the implementation of the buttons in `ActionButton` the same as those in `PrimaryTools`.
